resources/chefdk/msi/assets/start-chefdk.ps1

```diff
@@ -0,0 +1,22 @@
+$conemulocation = "$env:programfiles\ConEmu\Conemu64.exe"
+$chefdkinit = 'chef shell-init powershell | iex; Import-Module chef; $env:CHEFDK_ENV_FIX = 1'
+$chefdkgreeting = "echo 'PowerShell $($PSVersionTable.psversion.tostring()) ($([System.Environment]::OSVersion.VersionString))';write-host -foregroundcolor darkyellow 'Ohai, welcome to Chef-DK!`n'"
+$chefdkcommand = "$chefdkinit;$chefdkgreeting"
+$identity = [Security.Principal.WindowsIdentity]::GetCurrent()
+$principal = [Security.Principal.WindowsPrincipal] $identity
+$titleprefix = ""
+if($principal.IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator"))
+{
+    $titleprefix = "Administrator: "
+}
+
+$chefdktitle = "$($titleprefix)Chef-DK ($env:username)"
```

Comment:
Do we use the '-' in `Chef-DK` or not? Seems like not at the page below:
https://downloads.chef.io/chef-dk/
> The Chef Development Kit (ChefDK) brings the best-of-breed 

So we should change it I think to `ChefDK`.